src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.hpp

```diff
@@ -63,13 +63,28 @@ class ReconstructionEngine_sequentialSfM : public ReconstructionEngine
     bool filterTrackForks = true;
     robustEstimation::ERobustEstimator localizerEstimator = robustEstimation::ERobustEstimator::ACRANSAC;
     double localizerEstimatorError = std::numeric_limits<double>::infinity();
-    size_t localizerEstimatorMaxIterations = 4096;
+    std::size_t localizerEstimatorMaxIterations = 4096;
 
     // Pyramid scoring
 
     const int pyramidBase = 2;
     const int pyramidDepth = 5;
 
+    /// Bundle Adjustment triggering
+
+    // The beginning of the incremental SfM is a well known risky and
```

Comment:
Could be "///" to become part of the variable doxygen documentation.

Reply:
Done.